packages/odyssey-storybook/src/components/Box/Box.stories.tsx

```diff
@@ -12,16 +12,16 @@
 
 import React, { ReactElement } from "react";
 import { Story } from "@storybook/react";
-import { Source } from "./Box.docgen";
-import { Box, BoxProps, Heading, Text } from "@okta/odyssey-react";
+import { BoxDocGen } from "./Box.docgen";
+import { Box, BoxProps, Heading, Text } from "../../../../odyssey-react/src";
 
 import BoxMdx from "./Box.mdx";
 
-import "./box-stories.css";
+import "!style-loader!css-loader!./box-stories.css";
```

Comment:
I thought the custom babel-loader took care of this for us. Why do we have to inline Webpack compilation steps?